opencti-platform/opencti-front/src/private/components/analyses/external_references/ExternalReferencePopoverDeletion.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FunctionComponent, useState } from 'react';
+import { graphql } from 'react-relay';
+import Alert from '@mui/material/Alert';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import { useNavigate } from 'react-router-dom';
+import Security from '../../../../utils/Security';
+import { KNOWLEDGE_KNUPDATE_KNDELETE } from '../../../../utils/hooks/useGranted';
+import { useFormatter } from '../../../../components/i18n';
+import { deleteNodeFromId } from '../../../../utils/store';
+import Transition from '../../../../components/Transition';
+import useApiMutation from '../../../../utils/hooks/useApiMutation';
+
+export const externalReferencePopoverDeletionDeleteMutation = graphql`
+  mutation ExternalReferencePopoverDeletionDeleteMutation($id: ID!) {
+    externalReferenceEdit(id: $id) {
+      delete
+    }
+  }
+`;
+
+interface ExternalReferencePopoverDeletionProps {
+  id: string;
+  objectId?: string;
+  handleRemove: (() => void) | undefined;
+  isExternalReferenceAttachment?: boolean;
+}
+
+const ExternalReferencePopoverDeletion: FunctionComponent<
+ExternalReferencePopoverDeletionProps
+> = ({ id, objectId, handleRemove, isExternalReferenceAttachment }) => {
+  const { t_i18n } = useFormatter();
+  const navigate = useNavigate();
+  const [displayDelete, setDisplayDelete] = useState(false);
+  const [deleting, setDeleting] = useState(false);
+  const deleteSuccessMessage = t_i18n('', {
+    id: '... successfully deleted',
+    values: { entity_type: t_i18n('entity_External-Reference') },
+  });
+  const [commit] = useApiMutation(
+    externalReferencePopoverDeletionDeleteMutation,
+    undefined,
+    { successMessage: deleteSuccessMessage },
+  );
+  const handleOpenDelete = () => {
+    setDisplayDelete(true);
+  };
+  const handleCloseDelete = () => {
+    setDisplayDelete(false);
+  };
+  const submitDelete = () => {
+    setDeleting(true);
+    commit({
+      variables: {
+        id,
+      },
+      updater: (store) => {
+        if (handleRemove && objectId) {
+          deleteNodeFromId(
+            store,
+            objectId,
+            'Pagination_externalReferences',
+            undefined,
+            id,
+          );
+        }
+      },
+      onCompleted: () => {
+        setDeleting(false);
+        if (handleRemove) {
+          handleCloseDelete();
+        } else {
+          navigate('/dashboard/analyses/external_references');
+        }
+      },
+    });
+  };
+  return (
+    <React.Fragment>
```

Comment:
we prefer `<>`  (well it is the same but for reaseon of "uniformity")
```suggestion
    <>
```

Reply:
@marieflorescontact - Fixed/Updated.